+neuron/Vector.m

```diff
@@ -71,6 +71,23 @@
         %   value = length()
             value = self.call_method_hoc('size', 'double');
         end
+        
+        function value = size(self)
+        % Get Vector size.
+        %   value = size()
+            value = [1 self.length()];
```

Comment:
Do we have a document somewhere of where things behave differently between MATLAB and the other languages? If so, we should create one and this should be on it.

Because in Python and HOC, there's also a `Vector.size()` but there it returns a scalar. 

https://nrn.readthedocs.io/en/8.2.2/python/programming/math/vector.html#Vector.size

Reply:
Added section to readme.